recipes/dfp/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import copy, get
+
+required_conan_version = ">=1.54.0"
+
+
+class DfpConan(ConanFile):
+    name = "dfp"
+    package_type = "library"
+
+    # Optional metadata
+    description = "Decimal Floating Point Arithmetic Library"
+    homepage = "https://github.com/epam/DFP/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ("Apache-2.0", "Intel")
+    topics = ("decimal", "dfp", "ieee-754", "deltix")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+
+    # Sources are located in the same place as this recipe, copy them to the recipe
+    exports_sources = "CMakeLists.txt", "src/*", "include/*"
```

Comment:
This line is doing nothing - those files are not in this folder

Reply:
OK, I'll delete this line.